lib/utils.js

```diff
@@ -30,8 +31,20 @@ function normalizeBinary (binaryPath, platform, arch) {
                /linux/i.test(platform) ? "linux" + arch :
                platform;
 
+    var channelNames = ["firefox", "firefoxdeveloperedition", "beta", "nightly", "aurora"];
+
     var app = binaryPath.toLowerCase();
-    binaryPath = normalizeBinary.paths[app + " on " + platform] ||
+    var result = null;
+    if (platform === "osx" && channelNames.indexOf(binaryPath) !== -1) {
+      // Try to find the app for this channel.
+      // mdfind "kMDItemCFBundleIdentifier == 'org.mozilla.firefoxdeveloperedition'"
+      var results = spawnSync("mdfind", ["kMDItemCFBundleIdentifier == 'org.mozilla." + binaryPath + "'"]);
+      if (results.stdout) {
+        result = results.stdout.toString().split('\n')[0];
+      }
+    }
+    binaryPath = result ||
+                 normalizeBinary.paths[app + " on " + platform] ||
```

Comment:
Using a natural language key, ":alias on :platform", reminds of how perl hackers write code :eyes: